.nuspec/proguard.cfg

```diff
@@ -1,2 +1,4 @@
 -keep class android.support.v7.widget.FitWindowsFrameLayout { *; }
--dontwarn android.support.v7.widget.FitWindowsFrameLayout
\ No newline at end of file
+-dontwarn android.support.v7.widget.FitWindowsFrameLayout
+-keep class android.support.design.** { *; }
```

Comment:
If you get CI in place to check these rules are working, it might be worth it to get more specific and not use `**`? If you could run XF's UITest suite with proguard+release, that would help this case.

proguard/r8 can probably strip out more code if you listed individual classes.

You can also do this to allow non-public members to get stripped:
```
-keep class android.support.design.** { public *; }
```

This is all trial and error though, you will have to run a release app with proguard on to see if you get any java missing-method exceptions at runtime.

Reply:
Yea once we get this in place
https://github.com/xamarin/Xamarin.Forms/issues/5656

We'll look at refining this down